backend/src/api/routes/opportunities-route.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { run } from '~/common/helpers/route.helper';
+import {
+  getOpportunities,
+  createOpportunities,
+} from '~/services/opportunity.service';
+
+const router: Router = Router();
+router.get(
+  '/',
+  run((_) => getOpportunities()),
+);
+router.post(
+  '/',
+  run((req) => createOpportunities(req.body.opportunities)),
```

Comment:
use companyId from request to create opportunities under the needed company
and userId as well